docs/source/app-dev/app-arch.rst

```diff
@@ -189,21 +189,22 @@ new endpoint, it will resume normal operation.
 Dealing with time
 *****************
 
-The DAML language contains a function :ref:`getTime <daml-ref-gettime>` which returns the “current time”. The notion of time comes with a lot of problems in a distributed setting: different participants might run slightly different clocks, transactions would not be allowed to “overtake” each other during DAML interpretation, i.e., a long-running command could block all other commands, and many more.
+The DAML language contains a function :ref:`getTime <daml-ref-gettime>` which returns a rough estimate of “current time” called *Ledger Time*. The notion of time comes with a lot of problems in a distributed setting: different participants might run different clocks, there may be latencies due to calculation and network, clocks may drift against each other over time, etc.
 
-To avoid such problems, DAML provides the following concept of *ledger time*:
+In order to provide a useful notion of time in DAML without incurring severe performance or liveness penalties, DAML has two notions of time: *Ledger Time* and *Record Time*:
 
-- As part of command interpretation, each transaction is automatically assigned a *ledger time* by the participant server.
-- All calls to ``getTime`` within a transaction return the *ledger time* assigned to that transaction.
-- *Ledger time* is reasonably close to real time. To avoid transactions being rejected because the assigned *ledger time* does not match the ledger's system time exactly, DAML Ledgers define a tolerance interval around its system time. The system time is part of the ledger synchronization/consensus protocol, but is not known by the participant node at interpretation time. Transactions with a *ledger time* outside this tolerance interval will be rejected.
-- *Ledger time* respects causal monotonicity: if a transaction ``x`` uses a contract created in another transaction ``y``, transaction ``x``\ s ledger time will be greater than or equal to the ledger time of the referenced transaction ``y``.
+- As part of command interpretation, each transaction is automatically assigned a *Ledger Time* by the participant server.
+- All calls to ``getTime`` within a transaction return the *Ledger Time* assigned to that transaction.
+- *Ledger Time* is chosen (and validated) to respect Causal Monotonicity: The Create action on a contract *c* always precedes all other actions on *c* in Ledger Time.
```

Comment:
```suggestion
- *Ledger Time* is chosen (and validated) to respect Causal Monotonicity: The Create action on a contract *c* always precedes (or is equal to) all other actions on *c* in Ledger Time.
```

Reply:
I'm not sure that clarifies things. Plus I'd argue that if "all other actions" is the empty set, the statement is true as well.

Reply:
The statement before clearly had a `<=` relationship. `precedes` is somewhat ambiguous in whether you force strict inequality or not.